test/org/sagebionetworks/bridge/validators/ConsentSignatureValidatorTest.java

```diff
@@ -0,0 +1,240 @@
+                "   \"imageData\":\"\",\n" +
+                "   \"imageMimeType\":\"image/fake\"\n" +
+                "}";
+        ConsentSignature sig = BridgeObjectMapper.get().readValue(jsonStr, ConsentSignature.class);
+        assertValidatorMessage(validator, sig, "imageData", "cannot be an empty string");
+    }
+
+    @Test
+    public void jsonEmptyImageMimeType() throws Exception {
+        String jsonStr = "{\n" +
+                "   \"name\":\"test name\",\n" +
+                "   \"birthdate\":\"1970-01-01\",\n" +
+                "   \"imageData\":\"" + TestConstants.DUMMY_IMAGE_DATA + "\",\n" +
+                "   \"imageMimeType\":\"\"\n" +
+                "}";
+        ConsentSignature sig = BridgeObjectMapper.get().readValue(jsonStr, ConsentSignature.class);
+        assertValidatorMessage(validator, sig, "imageMimeType", "cannot be an empty string");
+    }
+
+    @Test
+    public void jsonImageDataWithoutMimeType() throws Exception {
+        String jsonStr = "{\n" +
+                "   \"name\":\"test name\",\n" +
+                "   \"birthdate\":\"1970-01-01\",\n" +
+                "   \"imageData\":\"" + TestConstants.DUMMY_IMAGE_DATA + "\"\n" +
+                "}";
+        ConsentSignature sig = BridgeObjectMapper.get().readValue(jsonStr, ConsentSignature.class);
+        try {
+            Validate.entityThrowingException(validator, sig);
+            fail("Should have thrown exception");
+        } catch(InvalidEntityException e) {
+            assertTrue(e.getMessage().contains(
+                    "ConsentSignature must specify imageData and imageMimeType if you specify either of them"));
+        }
+    }
+
+    @Test
+    public void jsonImageMimeTypeWithoutData() throws Exception {
+        String jsonStr = "{\n" +
+                "   \"name\":\"test name\",\n" +
+                "   \"birthdate\":\"1970-01-01\",\n" +
+                "   \"imageMimeType\":\"image/fake\"\n" +
+                "}";
+        ConsentSignature sig = BridgeObjectMapper.get().readValue(jsonStr, ConsentSignature.class);
+        try {
+            Validate.entityThrowingException(validator, sig);
+            fail("Should have thrown exception");
+        } catch(InvalidEntityException e) {
+            assertTrue(e.getMessage().contains(
+                    "ConsentSignature must specify imageData and imageMimeType if you specify either of them"));
+        }
+    }
+
+    @Test
+    public void minAgeLimitButNoBirthdate() {
+        validator = new ConsentSignatureValidator(18);
+        ConsentSignature sig = new ConsentSignature.Builder().withName("test name").build();
+        assertValidatorMessage(validator, sig, "birthdate", "cannot be missing, null, or blank");
+    }
+    
+    @Test
+    public void minAgeLimitButBirthdateTooRecent() {
+        String birthdate = DateTime.now().minusYears(18).plusDays(1).toLocalDate().toString();
+        validator = new ConsentSignatureValidator(18);
```

Comment:
Are we sure we have the timezone correct? The fixed time is 2022-02-28T00:00Z, which is 2022-02-27 in the local time zone, so we never have to worry about daylight savings time. However, this test will fail anywhere east of UTC.

I tested this with the old ConsentServiceTest (by fixing the date to 2022-02-28T00:00-0800, and changing it to plus/minus 1 day), and the test fails. I'm pretty sure this test will fail as well.